index.html

```diff
@@ -0,0 +1,56 @@
+<!DOCTYPE html>
+<html>
+    <head>
+        <link rel="stylesheet" href="styles.css">
+    </head>
+<body>
+<div class="all">
+    <header>
+        <img src="./images/My Logo.png" alt="logo holder">
```

Comment:
[OPTIONAL] I  suggest that for this you put it in a `<a>` link tag. Logos should be links  for three reasons: convention, semantics, and user experience(UX). [Check this link](https://ux.stackexchange.com/questions/81727/why-is-it-standard-for-a-website-logo-to-navigate-to-the-home-page) for more info.